upi/openstack/021_network.yaml

```diff
@@ -0,0 +1,40 @@
+- import_playbook: common.yaml
```

Comment:
same as before.  you could remove all of the "When: os_networking_type == "CiscoACI"  and simply include this file in 02_network.yaml with a when clause: 

like so: 
- name: Configure CiscoACI network
  incluce: 021_network.yaml     <--I woudl also change name to show it is for ciscoACI 
  when: os_networking_type == "CiscoACI"


saves typing all those when, and its easier to review and matain.